rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -35,6 +35,30 @@ foreach(_idl_file ${rosidl_generate_interfaces_IDL_FILES})
   endif()
 endforeach()
 
+if(NOT "${_generated_msg_files} " STREQUAL " ")
+  list(GET _generated_msg_files 0 _msg_file)
+  get_filename_component(_parent_folder "${_msg_file}" DIRECTORY)
+  list(APPEND _generated_msg_files
+    "${_parent_folder}/__init__.py"
+  )
+  get_filename_component(_parent_folder "${_parent_folder}" DIRECTORY)
+  list(APPEND _generated_msg_files
+    "${_parent_folder}/__init__.py"
```

Comment:
This adds two different `__init__.py` files but the install rule only has one destination. I don't think the files will be installed in the location you expect them to?